client/app/bundles/course/assessment/pages/AssessmentStatistics/SubmissionTimeAndGradeStatistics/SubmissionTimeAndGradeChart.tsx

```diff
@@ -69,49 +65,42 @@ const SubmissionTimeAndGradeChart = ({ submissions, intl }) => {
     ],
   };
 
-  const hasEndAt = submissions.every((s) => s.endAt != null);
```

Comment:
done.. I have done the reconsideration for this code snippet, since previously I thought the submissions here actually belongs to different type. But it can be mitigated well here and hence I put it back here